signingscript/requirements/base.in

```diff
@@ -1,10 +1,10 @@
 arrow
 # TODO: Remove the following line once flake8, jsonschema, pep517, pluggy, pytest,
 # tox, and virtualenv all support importlib-metadata >= 2
-importlib-metadata<2; python_version < '3.8'
+importlib-metadata<2;
```

Comment:
Does this mean we need `importlib-metadata` for py38 and py39 now?

The test failure is because we updated `base.in` but we haven't updated the `*.txt` requirements files.
We need to run [pin.sh](https://github.com/mozilla-releng/scriptworker-scripts/blob/master/maintenance/pin.sh). If you want to limit it to just signingscript, you can `./pin.sh signingscript` .